### ISSUE TITLE
Suggestion: make features filterable by HTML/CSS/JS/etc. 

### ISSUE THREAD

**Issue by SachaG** [web-platform-dx/web-features-explorer#21]
I was browsing https://web-platform-dx.github.io/web-features-explorer/recent/, and it would be quite useful to be able to only show CSS features, or maybe just HTML+JS.

**Comment by captainbrosset** [comment]
I agree with this. My hope is that we make web-features groups better, and use them to filter the various pages on the explorer.
You can see the groups we currently have on this in-progress page: https://web-platform-dx.github.io/web-features-explorer/groups/

As you can see, not all features are grouped yet. More importantly, we don't really know how we want to group yet. We discussed this on 9/5 ([notes](https://docs.google.com/document/d/1ree75ImLZjf60lTZ3BhCaLHygxgywr7SBXp-q0xPs8A/edit#heading=h.f4j2gvqnbhp9)). We know they're useful to us right now, and might also become very useful to others (State Of, MDN) in the future. That said, there are many possible ways to group, and only consumers will really know how they want to group.
I think there was some agreement that web-features should still define _a_ default grouping, even if other consumers decide to group differently over time.

That said, let's assume for a minute that all features are grouped, and all our groups are amazing. How would we want to filter the /recent/ (and other) page based on them? I'm imagining a sidebar that shows a treeview representation of the group hierarchy. Clicking on any of the nodes in the treeview would filter features that are in this group.

@SachaG thoughts?

**Comment by SachaG** [comment]
Oh right, I forgot about groups. For my purposes I think I would simply need to be able to filter by JS/CSS/HTML, at least for now. Although more complex filtering would also be cool. Can a feature belong to more than one group by the way?

**Comment by captainbrosset** [comment]
I know this isn't a filter yet, but just to show progress, I added the first part of BCD key paths and groups to the explorer. Here's what it looks like for now:
![image](https://github.com/user-attachments/assets/49dc2b00-4605-433a-8e26-d573d6c20ddd)

I'm imagining that clicking on these tags/groups would filter the page to only show the features that belong to the same tag or group.
